README.md

```diff
@@ -22,6 +22,61 @@ It can be used to scaffold high-quality draft genome assemblies with any long
 sequences (eg. ONT reads, PacBio reads, other draft genomes, etc).
 It is also used to scaffold contig pairs linked by ARCS/ARKS.
 
+### Dependencies
+-------------------------------
+
+- GCC (tested on v7.2.0)
+- Perl (tested on v5.32.1)
+- Autotools (if cloning directly from repository)
+
+### Compilation:
+-------------------------------
+If cloning directly from the repository run:
+
+```
+cd releases/links_v2.0.0/
+git submodule init
+git submodule update
+```
+
+Generate autotools scripts:
+
+```
+./autogen.sh
+```
+
+To compile LINKS run:
+
+```
+./configure && make
+```
+To install LINKS in a specified directory:
+
+```
+./configure --prefix=/LINKS/PATH && make install
+```
+* *These steps worked on a CentOS 7 system with 128 CPU Intel(R) Xeon(R) CPU E7-8867 v3 @ 2.50GHz
+
+### Running LINKS v2.0.0
+-------------------------------
+Users can run `LINKS` (shell script) to run whole pipeline smoothly with LINKS naming convention(such as -d xx -t yy) as pre 2.0.0 versions.
+
+To run LINKS with default parameters with `LINKS`:
+```
+./releases/links_v2.0.0/LINKS -f NA1281_draft.fa -s NA1281_reads.fof
+```
+
+* *LINKS v2.0.0 is implemented in C++ and perl. The C++ executable is `src/LINKS_CPP` and PERL executable is `src/LINKS.pl`. To run the full LINKS pipeline smoothly you can run `LINKS-make`. However `LINKS-make` is a Makefile and have different argument naming convention(such as d=xx t=yy instead of -d xx -t yy).
```

Comment:
```suggestion
* *LINKS v2.0.0 is implemented in C++ and perl. The C++ executable is `src/LINKS_CPP` and PERL executable is `src/LINKS.pl`. To run the full LINKS pipeline smoothly you can run `LINKS-make`. However `LINKS-make` is a Makefile and has a different argument value assignment (such as d=xx t=yy instead of -d xx -t yy).
```